zim/notebook/index/pages.py

```diff
@@ -539,6 +539,17 @@ def walk_bottomup(self, path=None):
 		page_id = self._pages.get_page_id(path) if path else ROOT_ID # can raise
 		return self._pages.walk_bottomup(page_id)
 
+	def search_pagename_substring(self, nameFragment):
```

Comment:
For consistency in the class I would suggest calling this `list_pages_match_substring(self, substring)`